### ISSUE TITLE
Upercasing first letter and numeric data issues

### ISSUE THREAD

**Issue by n9yty** [englishextra/qrjs2#11]
* [x] My **browser** is:  Chrome, Microsoft Edge (Mac) and Safari

* [x] I'm using version [v0.1.11]

* [x] I am sure this issue is **not a duplicate**?  Yes

### Description

I am not sure what is going on, but I am trying to incorporate this into my project in order to generate QR codes to print on labels using a product SKU (alphanumeric/symbols) and a barcode number. I found that I had to convert the barcode number into a string for it to work, otherwise it complained: Uncaught too large data @ qrjs2.js:803. I don't know if this is normal or if I am just not specifying something properly.

I didn't know QR could not encode lowercase letters, so that is an issue we have to work through. :)

Are there some options I am not passing in? I am using very basics from the sample:

`{
  ecclevel: "M",
  fillcolor: "#FFFFFF",
  textcolor: "#000000",
  margin: 4,
  modulesize: 8
}`

**Comment by englishextra** [comment]
@n9yty Could you please provide your SKU (or whatever it was) string that you pass to text option

I tested 

```js
				var div = document.createElement("div"),
				text = "samsgordon-rd-9.5",
				qr = QRCode.generateSVG(text, {
						ecclevel: "M",
						fillcolor: "#F2F2F2",
						textcolor: "#D13438",
						margin: 4,
						modulesize: 8
					});
				div.appendChild(qr);
				document.body.appendChild(div);
```

and it worked


![2021-10-01_000359](https://user-images.githubusercontent.com/537127/135530752-6ca4f081-454e-436e-ad11-541a43de4fb6.jpg)

**Comment by n9yty** [comment]
Yes, the problem was my mistake about the uppercasing, the application I was scanning into seems to have been doing that. I had no idea. I also found that lowercase is encoded using OCTET type instead of alphanumeric.

The only question then is the error when passing in the barcode as a number.  It is not a problem to convert to string first, it just is not necessarily expected behavior and the error was not clear.

Sorry for the confusion. It is nice to see a JS QR project that is updated, the other popular one I found has not been touched in a very long time.
